docs/site/sidebars/lb4_sidebar.yml

```
@@ -334,61 +371,32 @@ children:
         url: Appsody-LoopBack.html
         output: 'web, pdf'
 
-  - title: 'Using Express Middleware'
-    url: Express-middleware.html
-    output: 'web, pdf'
-
-  - title: 'Serving Static Files'
-    url: Serving-static-files.html
-    output: 'web, pdf'
-
-  - title: 'Dynamically adding models, repositories, and controllers'
-    url: Dynamic-models-repositories-controllers.html
-    output: 'web, pdf'
-
-  - title: 'Uploading and Downloading Files'
-    url: File-upload-download.html
-    output: 'web, pdf'
-
-  - title: 'Validation'
-    url: Validation.html
+  - title: 'Running and Debugging Apps'
```

I find this section a bit weird. We already have "Deployment" that's covering many aspects of "running apps". If we move strong-error-handler page to REST APIs, then we get a single page "setting debug string" (for now).

I am proposing to rename this section to "Troubleshooting", to make the scope wider than just debugging, but at the same time to make it clear what kind of content it contain (in which scenarios should users look here).

```suggestion
  - title: 'Troubleshooting'
```

I have already few ideas on more content (new pages) to add here, e.g. how to troubleshoot "Binding XYZ not found" problem.